### PR TITLE
[WIFI-4648] Add: script to manually create release

### PR DESCRIPTION
**Pull request by dunaev-opsfleet**

### CODE REVIEW

File: .github/git-release-tool/git-release-tool.sh [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+## Git manipulation functions
+modify_deploy_repo_values() {
+  NEW_RELEASE_TAG=$1
+  REPOSITORIES_AMOUNT=$(cat ../repositories.yaml | yq ".repositories[].name" -r | wc -l)
+  for REPO_INDEX in $(seq 0 $(expr $REPOSITORIES_AMOUNT - 1)); do
+    REPO_URL=$(cat ../repositories.yaml | yq ".repositories[$REPO_INDEX].url" -r)
+    REPO_URL_WITHOUT_SUFFIX=${REPO_URL%.git}
+    REPO_DOCKER_COMPOSE_NAME=$(cat ../repositories.yaml | yq ".repositories[$REPO_INDEX].docker_compose_name" -r)
+    SERVICE_TAG="${REPO_TAGS_ARRAY[$REPO_INDEX]}"
+    sed "s/$REPO_DOCKER_COMPOSE_NAME=.*/$REPO_DOCKER_COMPOSE_NAME=$SERVICE_TAG/" -i docker-compose/.env
```
Comment (oblom0v) [inline .github/git-release-tool/git-release-tool.sh:94]: there are two more files which have to be adapted, `.env.letsencrypt` and `.env.selfsigned`.

File: .github/workflows/git-release.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - name: Run release creation script
+      working-directory: wlan-cloud-ucentral-deploy/.github/git-release-tool
+      run: |
+        export RELEASE_VERSION=${{ github.event.inputs.release_version }}
+        export TAG_TYPE=${{ github.event.inputs.tag_type }}
+        export GIT_PUSH_CONFIRMED=${{ github.event.inputs.push_confirm }}
+        export LOG_VERBOSITY=${{ github.event.inputs.log_level }}
+        git config --global credential.helper store
+        git config --global user.email "dmitry@opsfleet.com"
+        git config --global user.name "Dmitry Dunaev"
```
Comment (oblom0v) [inline .github/workflows/git-release.yml:50]: I think we should switch to the tip-automation-user here so it's evident that the changes were made by an automated process.

File: .github/git-release-tool/git-release-tool.sh (hunk 1 above)
Comment (dunaev-opsfleet) [reply .github/git-release-tool/git-release-tool.sh:94]: Sure, thanks for noticing!

File: .github/workflows/git-release.yml (hunk 2 above)
Comment (dunaev-opsfleet) [reply .github/workflows/git-release.yml:50]: Do we have one that we are using at the moment?

Comment (oblom0v) [reply .github/workflows/git-release.yml:50]: I guess https://github.com/tip-automation could be used for this case.

Comment (dunaev-opsfleet) [reply .github/workflows/git-release.yml]: Updated

**Review by oblom0v — APPROVED** [review]

**Pull request merged**